tbx/blog/tests/test_models.py

```diff
@@ -59,3 +63,31 @@ def test_create(self):
         )
         self.assertEqual(another_blog_post.related_services.count(), 3)
         self.assertEqual(another_blog_post.related_sectors.count(), 2)
+
+    def test_related_blog_posts_same_division(self):
+        sector = SectorFactory.create()
+        # The logic of `final_division` skips ancestors with depth <= 2,
+        # So we create the division page with enough parents to be at depth 3:
+        division = DivisionPageFactory.create(parent=PageFactory(parent=PageFactory()))
```

Comment:
(info, just fyi) We usually stick pages under the HomePage. On other projects, the homepage would already be generated in tests because of the migrations, but it looks like this isn't the case on torchbox.com

Reply:
Thanks for the info 👍🏻